sender/influxdb.go

```diff
@@ -58,7 +59,26 @@ func (idb *InfluxDB) Send(c *skogul.Container) error {
 		idb.client = &http.Client{Timeout: idb.Timeout.Duration}
 	})
 	for _, m := range c.Metrics {
-		fmt.Fprintf(&buffer, "%s", idb.Measurement)
+		measurement := idb.Measurement
+		if idb.MeasurementFromMetadata != "" {
+			measure, ok := m.Metadata[idb.MeasurementFromMetadata].(string)
+			if ok {
+				measurement = measure
+			}
+			// The reason this isn't an else-if is because now
+			// it also catches the scenario where the type cast
+			// is successful, but the key is empty.
+			if measurement == "" {
+				// XXX:
+				// How do we report issues of single
+				// metrics failing, but not the container
+				// in general? Failing the entire container
+				// for just one failed metric is not really
+				// acceptable...
+				continue
```

Comment:
This could be configured by the end user. Is a metric failing transform any relevant? If yes: use it, if no: discard it. This was implemented in the Split transformer. Can be left as a "todo" (new GH issue?) for now.

Reply:
Makes a certain amount of sense, I'll open a separate issue on it, because I'm not coming up with a good name for it right this moment.